src/main/kotlin/no/nav/dings/token/tokendings/TokenDingsService.kt

```diff
@@ -35,13 +35,13 @@ internal const val BEARER = "Bearer"
 class TokenDingsService(
     private val tokenDingsConfig: Environment.TokenDings
 ) {
-    private val jwkToRSA = JWKSet.parse(tokenDingsConfig.jwksPrivate).keys[0].toRSAKey()
+    private val jwkToRSA = RSAKey.parse(tokenDingsConfig.tokenXPrivateJwk)
 
     @KtorExperimentalAPI
     fun clientAssertion(): String {
-        log.info { "Getting Keys with keyIDs: ${JWKSet.parse(tokenDingsConfig.jwksPrivate).keys.map { it.keyID }}" }
+        log.info { "Getting Keys with keyIDs: ${JWKSet.parse(tokenDingsConfig.tokenXPrivateJwk).keys.map { it.keyID }}" }
```

Comment:
It is probably redundant to parse this into a JWKSet, use the keyID from the jwkToRSA directly?